src/Core/Tokens/SsoRedirectToken.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Linq;
+
+namespace Bit.Core.Tokens
+{
+    public static class SsoRedirectToken
+    {
+        #region Constants
+
+        public const string SSO_REDIRECT_COOKIE_NAME    = "SsoRedirect";
+        private const int MAX_REDIRECT_COOKIE_MINUTES   = 5;
```

Comment:
5 minutes is too long, from preValidate() to Login() there shouldn't be more than a 30 second delay, I would say 1 minute to be safe on a really, really, really slow 3G LTE connection out in the middle of nowhere, but that would be the max we would want this token being valid for, to prevent replays as much as possible.